x-pack/plugins/apm/server/routes/fleet.ts

```diff
@@ -170,15 +172,34 @@ const createCloudApmPackagePolicyRoute = createApmServerRoute({
     if (!hasRequiredRole || !cloudApmMigrationEnabled) {
       throw Boom.forbidden(CLOUD_SUPERUSER_REQUIRED_MESSAGE);
     }
-    return {
-      cloud_apm_package_policy: await createCloudApmPackgePolicy({
-        cloudPluginSetup,
-        fleetPluginStart,
-        savedObjectsClient,
-        esClient,
-        logger,
-      }),
-    };
+
+    const cloudApmAackagePolicy = await createCloudApmPackgePolicy({
+      cloudPluginSetup,
+      fleetPluginStart,
+      savedObjectsClient,
+      esClient,
+      logger,
+    });
+
+    const [setup, internalSavedObjectsClient] = await Promise.all([
+      setupRequest(resources),
+      core
+        .start()
+        .then(({ savedObjects }) => savedObjects.createInternalRepository()),
+    ]);
+
+    const spaceId = plugins.spaces?.setup.spacesService.getSpaceId(request);
+
+    // force update the index pattern title with data streams
+    await createStaticIndexPattern(
+      setup,
+      config,
+      internalSavedObjectsClient,
+      spaceId,
+      true
+    );
+
+    return { cloud_apm_package_policy: cloudApmAackagePolicy };
```

Comment:
nit: Any reason this is in snake case?

Reply:
Especially considering we are doing the opposite transformation on the clientside this feels unnecessary: 

https://github.com/elastic/kibana/blob/1739b55f94df04b4b99fbc11233cc16d5cef863b/x-pack/plugins/apm/public/components/app/Settings/schema/index.tsx#L145-L150